queue_services/entity-emailer/src/entity_emailer/config.py

```diff
@@ -20,131 +40,74 @@
 or by accessing this configuration directly.
 """
 import os
-import random
 
 from dotenv import find_dotenv, load_dotenv
 
 
 # this will load all the envars from a .env file located in the project root (api)
 load_dotenv(find_dotenv())
 
-CONFIGURATION = {
-    'development': 'legal_api.config.DevConfig',
-    'testing': 'legal_api.config.TestConfig',
-    'production': 'legal_api.config.ProdConfig',
-    'default': 'legal_api.config.ProdConfig'
-}
-
-
-def get_named_config(config_name: str = 'production'):
-    """Return the configuration object based on the name.
-
-    :raise: KeyError: if an unknown configuration is requested
-    """
-    if config_name in ['production', 'staging', 'default']:
-        config = ProdConfig()
-    elif config_name == 'testing':
-        config = TestConfig()
-    elif config_name == 'development':
-        config = DevConfig()
-    else:
-        raise KeyError(f'Unknown configuration: {config_name}')
-    return config
-
 
-class _Config():  # pylint: disable=too-few-public-methods
+class Config:  # pylint: disable=too-few-public-methods
     """Base class configuration that should set reasonable defaults.
 
     Used as the base for all the other configurations.
     """
 
     PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
 
-    MSG_RETRY_NUM = int(os.getenv('MSG_RETRY_NUM', '5'))
+    MSG_RETRY_NUM = int(os.getenv("MSG_RETRY_NUM", "5"))
```

Comment:
Should I run the Black formatter on all files in entity-emailer, or should I undo the double quotes?

Reply:
Sounds good, I'll hold off on running the formatter for now just to keep the files changed more legible for reviews. I'll run it through the black formatter as the last thing I do before merging.